CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# How to contribute #
+
+All help is appreciated. All contributors are unpaid volunteers. If you really want to see something added to Exiv2 - volunteer!
+
+
+## General workflow for contributing ##
+
+1. Fork, then clone the repo:
+```
+git clone git@github.com:your-username/exiv2.git
+```
+
```

Comment:
That is standard for GitHub hosted repos and documented at https://guides.github.com/activities/forking/, for example.

Reply:
I could add that that link at the beginning of the section, but I think it is interesting anyways to keep these points as a quick reference workflow to contribute.